rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "1.54"
```

Comment:
I don't think I've seen a rust-toolchain file with libraries before. Normally you want to test against a range of Rust versions (MSRV, stable, beta, nightly).

Beta and Nightly are recommended to report  regressions back to the Rust team and to fix stuff like lints and future incompatibilities before they are released. Failures on beta/nightly shouldn't be a blocker for merging.

I see the appeal of writing shorter CI scripts but I am not sure it is worth it here? You are limiting yourself to only testing against a single Rust version.

Reply:
You're totally right, I'd never about all this before!